Controllers/ChatController.cs

```diff
@@ -21,12 +21,12 @@ namespace OnlineConsulting.Controllers
     public class ChatController : Controller
     {
         const int PAGE_SIZE = 10;
-        private readonly IChatRepository _chatRepository;
+        private readonly IConversationRepository _conversationRepository;
         private readonly IConfiguration _configuration;
 
-        public ChatController(IChatRepository chatRepository, IConfiguration configuration)
+        public ChatController(IConversationRepository conversationRepository, IConfiguration configuration)
         {
-            _chatRepository = chatRepository;
+            _conversationRepository = conversationRepository;
```

Comment:
You could add to every initialization line check if value is not null eg.
`_conversationRepository = conversationRepository ?? throw new ArgumentNullException(nameof(conversationReposotry));`